static/css/impala/users.less

```diff
@@ -360,3 +360,9 @@ img.icon {
         width: 100px;
     }
 }
+
+.notification-box.anonymous {
```

Comment:
I didn't want to mess with all of the error messages. I just noticed this one had wonky padding but I'm not sure if there are other ways to get an error message on this page that looks right. As a side note I'm not sure I've seen any of our error messages look right.